src/coriolis.jl

```diff
@@ -24,22 +24,59 @@ end
     FPlane([T=Float64;] f)
 
 Returns a parameter object for constant rotation at the angular frequency
-`2f`, and therefore with background vorticity `f`, around a vertical axis.
+`f`, and therefore with background vorticity `2f`, around a vertical axis.
```

Comment:
This is incorrect, because the vorticity is half the rotation rate. The rotation rate is `2f`, but the background vorticity is `f`.

Reply:
Oops. I was thrown off by the `fu`, `fv` functions and got confused.

Reply:
Shouldn't rotation rate be `f/2`?

Reply:
You're right, sorry about that. Rotation rate is `f/2`, vorticity is `f`.